README.md

```diff
@@ -215,3 +192,379 @@ if available(.settingsFeature) {
     )
 }
 ```
+
+## Code style
+
+The basis of the style of writing code is the configured Swiftlint in the project. The specific practices we follow are listed below.
+
+### Calls from View to ViewModel
+Name ViewModel functions as action from View:
+
+🟢
+```
+.onTapGesture {
+    viewModel.buttonTapped()
+}
+```
+
+🔴
+```
+.onTapGesture {
+    viewModel.sendToken()
+}
+```
+
+
+### Protocols
+Interfaces used for abstractions are named without prefixes and postfixes:
+
+🟢
+```
+protocol BuyService {}
+```
+
+🔴
+```
+protocol BuyServiceProtocol {}
+```
+
+
+For implementation or mocks we append `Impl`, `Mock` or other key words in the ending:
```

Comment:
consider to remove Impl from names

Reply:
We has already voted to choose Impl for implementation and protocol without suffix https://discord.com/channels/767761912167006292/900671311678955520/969520780037718046